MANUAL.txt

```diff
@@ -376,7 +376,7 @@ header when requesting a document from a URL:
     environment variable).  If that directory does not exist and
     `$HOME/.pandoc` exists, it will be used (for backwards compatibility).
     On Windows the default user data directory is
-    `C:\Users\USERNAME\AppData\Roaming\pandoc`.
+    `%APPDATA%\pandoc.
```

Comment:
Let's add the closing backtick after pandoc, and I think this would now fit on the previous line.